.pre-commit-config.yaml

```diff
@@ -42,6 +42,7 @@ repos:
                 docs/source-pytorch/_static/images/general/pl_quick_start_full_compressed.gif|
                 docs/source-pytorch/_static/images/general/pl_overview_flat.jpg|
                 docs/source-pytorch/_static/images/general/pl_overview.gif|
+                docs/source-apps/|
```

Comment:
I think you should list specific files not all folder

Reply:
negative... there are too many example scripts